src/ansys/geometry/core/geometry/parameterization.py

```diff
@@ -202,6 +231,139 @@ def get_span(self) -> Real:
 
         return self.end - self.start
 
+    def get_relative_val(self, t: Real) -> Real:
+        """
+        Return an evaluation property of the interval, used in BoxUV.
+
+        Parameters
+        ----------
+        t : Real
+            The offset that the interval gets evaluated at.
+
+        Returns
+        -------
+        Real
+            The actual value according to the offset
+        """
+        return self.start + t * self.get_span()
+
+    def is_negative(self, tolerance: Real) -> bool:
+        """
+        Boolean value that indicates whether the current interval is negative.
+
+        Parameters
+        ----------
+        tolerance : Real
+            The accepted range since we could be working with doubles
+
+        Returns
+        -------
+        bool
+            True if negative False otherwise
+        """
+        return Accuracy.compare_with_tolerance(self.get_span(), 0, tolerance, tolerance)
+
+    @staticmethod
+    def unite(first: "Interval", second: "Interval") -> "Interval":
+        """
+        Return the union of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The union of the two intervals
+        """
+        if first.is_empty():
+            return second
+        if second.is_empty():
+            return first
+        return Interval(min(first.start, second.start), max(first.end, second.end))
+
+    @staticmethod
+    def intersect(first: "Interval", second: "Interval", tolerance: Real) -> "Interval":
+        """
+        Return the intersection of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The intersection of the two intervals
```

Comment:
```suggestion
        Get the intersection of two intervals.

        Parameters
        ----------
        first : Interval
            First interval.
        second : Interval
            Second interval.

        Returns
        --------
        Interval
            Intersection of the two intervals.
```
I may have used "Find the intersecton...." for an earlier method. If so, use the same verbs for both.